pkg/eden/eden.go

```diff
@@ -33,6 +33,50 @@ type LinkState struct {
 	IsUP          bool
 }
 
+//StartSwtpm function run swtpm in docker
+func StartSwtpm(swtpmTag string) (err error) {
+	volumeMap := map[string]string{"/swtpm":"/tmp"}
```

Comment:
I don't think this will easily work on systems like MacOS and Windows -- not sure how big of a concern that is